### DISCUSSION TITLE
我也一样，是不是没按时提交

### DISCUSSION THREAD

**Discussion by LinePoints** [MlgmXyysd/Xiaomi-BootLoader-Questionnaire#87] [General]
我也一样，是不是没按时提交

_Originally posted by @windycold in https://github.com/MlgmXyysd/Xiaomi-BootLoader-Questionnaire/discussions/73#discussioncomment-8525876_

**Comment by xbn2002** [comment]
可能是超时了，没触发回调链接，问卷没有在社区登记判分
